src/Sentry/Internal/Http/CachingTransport.cs

```diff
@@ -65,34 +65,38 @@ public CachingTransport(ITransport innerTransport, SentryOptions options)
                 }
             }
 
-            _worker = Task.Run(async () =>
+            _worker = Task.Run(CachedTransportBackgroundTask);
+        }
+
+        private async Task? CachedTransportBackgroundTask()
```

Comment:
Method is easier to visualize in the stacktrace while debugging on rider.
Shouldn't return nullable but